$MirOS: ports/audio/libao/patches/patch-configure_in,v 1.1 2005/05/16 20:25:58 tg Exp $
--- configure.in.orig	Tue Jan 11 22:36:53 2005
+++ configure.in	Wed Nov  9 20:03:18 2005
@@ -173,6 +173,7 @@ dnl Check for ESD
 AC_ARG_ENABLE(esd, [  --enable-esd            include ESD output plugin ],
 [ BUILD_ESD="$enableval" ],[ BUILD_ESD="yes" ])
 
+have_esd=no
 if test "$BUILD_ESD" = "yes"; then
   AM_PATH_ESD(0.2.8, have_esd=yes, have_esd=no)
 fi
